dev/src/Command/SplitCommand.php

```diff
@@ -235,13 +244,15 @@ private function processComponent(
         $parentTagSource,
         $updateReleaseNotes,
         ?Packagist $packagist,
+        ?string $branchToPush = null
```

Comment:
super nit, but the `= null` makes the parameter optional, and we always pass a value (even if it's `null`), so this would more appropriately be simply
```suggestion
        ?string $branchToPush
```